src/core/main.c

```diff
@@ -1396,20 +1400,23 @@ static int become_shutdown(
 
         assert(pos < ELEMENTSOF(command_line));
 
-        if (STR_IN_SET(shutdown_verb, "reboot", "kexec") &&
-            arg_shutdown_watchdog > 0 &&
-            arg_shutdown_watchdog != USEC_INFINITY) {
+        if (streq(shutdown_verb, "reboot"))
+            watchdog_timer = arg_shutdown_watchdog;
+        else if (streq(shutdown_verb, "kexec"))
+            watchdog_timer = arg_kexec_watchdog;
```

Comment:
please stick to correct indentation, see https://systemd.io/CODING_STYLE

Reply:
Fixed